invesalius/gui/task_navigator.py

```diff
@@ -1563,14 +1656,16 @@ def OnSelectSize(self, evt, ctrl):
     def OnChangeCurrentSession(self, new_session_id):
         self.current_session = new_session_id
 
-    def CreateMarker(self, coord=None, colour=None, size=None, label='*', is_target=False, seed=None, session_id=None):
+    def CreateMarker(self, coord=None, colour=None, size=None, label='*', is_target=False, seed=None, head=None, robot=None, session_id=None):
```

Comment:
Do we really need the head location? Is this something that will be useful for all users or just for development? I would reconsider adding these unless it is strictly necessary.